package.json

```diff
@@ -23,7 +23,7 @@
     "build": "tsc",
     "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:docs": "markdownlint '**/*.md'",
-    "lint:js": "eslint --cache .",
+    "lint:js": "eslint --cache . --fix",
```

Comment:
The basic lint scripts should not automatically fix issues. The typical convention is to have a separate `lint:js:fix` script that would do the fixing.